src/App.tsx

```diff
@@ -10,29 +14,37 @@ import Footer from './components/footer';
 import ErrorBoundary from './components/ErrorBoundary';
 
 function App() {
-
   // Local State
   const [movieSelected, setMovieSelected] = React.useState({
```

Comment:
Personally i think that it be better to use `null` or `undefined` for empty movie here.
```js
const [movieSelected, setMovieSelected] = React.useState<Movie | null>(null);
```

Reply:
resolved